test/unit/spec/EnrollSms_spec.js

```diff
@@ -16,11 +16,12 @@ define([
   'helpers/xhr/MFA_ENROLL_ACTIVATE_success',
   'helpers/xhr/MFA_ENROLL_ACTIVATE_error',
   'helpers/xhr/MFA_ENROLL_ACTIVATE_errorActivate',
+  'helpers/xhr/MFA_ENROLL_ACTIVATE_phone_error.js',
```

Comment:
I think it will be better to rename that file to ```MFA_ENROLL_ACTIVATE_invalid_phone.js```, so we are a little more explicit on what the error is